community/governance.md.tmpl

```diff
@@ -37,14 +37,20 @@ Here are defined the primary teams participating in conda-forge activities.
   core, part of staged-recipes, or maintainers. This includes first-time
   contributors, collaborators, and funders. They have no special rights within
   the conda-forge organization itself.
-* **emeritus-core:** Core members that are inactive (commits, GitHub comments/issues/reviews,
-  dev meetings and voting on polls) in the past six months will be asked if they want to become emeritus-core
-  developers. Any core member can also request to become emeritus if they wish to do so
-  (e.g. taking a sabbatical or long vacation).
-  Emeritus core members can still vote and move back to active core anytime. Emeritus
-  votes are used to count towards quorum but the quorum size is computed from the size of
-  the active core group. The `core.csv` list should be updated when a change in the status
-  of a member occurs.
+* **emeritus-core:** Core members that are inactive (commits, GitHub
+  comments/issues/reviews, chat platforms, dev meetings, and voting on polls) in the
+  past six months will be asked if they want to become emeritus-core developers. One
+  week after asking, if the inactive core member has not responded, they will be
+  automatically moved to emeritus-core. Any core member can also request to become
+  emeritus if they wish to do so (e.g. taking a sabbatical or long vacation).
+  Every 12 months or more, one core member should ask the rest of the team to
+  report their activity status. The question must only include two answers:
+  active or inactive. Core members must have at least a week to reply. Members that reply
```

Comment:
I feel like a week is too little here.

Reply:
I like 2 weeks b/c that is a meeting cycle and we can try to get the member into the meeting and either refresh their participation or talk it out.

Reply:
Or a month. 2 meetings.

Reply:
```suggestion
  active or inactive. Core members must reply within four weeks. Members that reply
```

Reply:
A note on meetings: Due to timezone accommodations of our meeting (which are reasonable), it is not always possible to make a meeting every 2 weeks

Personally am able to go 1 meeting a month at this point (and maybe not if there was a conflict or I was out that week)

This may be even more true for folks who always deal with odd hours when trying to attend our meeting times

So I don't think meetings are the measure we should strongly require necessarily. Though I do think getting some face time periodically is a good thing and meetings serve an important purpose. Just am a little hesitant using them as a cutoff in this way

Reply:
We are not using meetings in any way here. A core member can simply reply to an email saying they are still active, or a github issue. They do not have to show up at a meeting to remain active and it doesn't say that anywhere in the text.

Reply:
That's fair

However the discussion above was using meetings to motivate the time window used with responses. Hence my comment